PowerPointLabs/PowerPointLabs/PowerPointLabs.csproj

```diff
@@ -285,7 +285,6 @@
     <Compile Include="TextCollection.cs" />
     <Compile Include="Utils\Common.cs" />
     <Compile Include="Utils\Comparers.cs" />
-    <Compile Include="Utils\Extensions.cs" />

```

Comment:
That change came from kai33, under commit c7e28a7 "make it compilable". It was on the master branch, so I cherry picked it onto the dev branch as well so that the dev branch would compile.